src/transformStateWithClones.js

```diff
@@ -7,7 +7,43 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const statesArray = [];
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
you can just use `for of`
```suggestion
   for (const action of actions)) {
```